### PR TITLE
Updated symbol kind mapping

### PR DESCRIPTION
**Pull request by alexsaveliev**
- returning 'variable' by default
- difference from VS code (enum members and constants are displayed as constants, not as variables)

### CODE REVIEW

**Comment by felixfbecker** [comment]
What is the difference to VS Code you mentioned?

**Comment by alexsaveliev** [comment]
difference (known) is for constants and enum members. VS code uses `variable`, we are using `constant` symbol kind. 

VS Code:
vscode-sym

Sourcegraph:
sg-sym

File: src/util.ts [hunk 1]
```diff
@@ -43,28 +43,39 @@ export function toUnixPath(filePath: string): string {
...
+		// case 'keyword'
+		// case 'type'
+		// case 'call'
+		// case 'index'
+		// case 'construct'
+		// case 'primitive type'
+		// case 'label'
+		// case 'directory'
+		// case 'external module name'
+		// case 'external module name'
```
Comment (alexsaveliev) [inline src/util.ts]: because there are textual constants in typescript code but no matching symbol kind (at least I can't think of matching one) in LSP

Comment (felixfbecker) [inline src/util.ts]: That's okay, but they should be mapped to _something_, so lets not comment out the cases

Comment (alexsaveliev) [inline src/util.ts]: they are mapped by the `default` statement to `variable`

**Pull request merged**